gheb/configuration-converter-bundle/0.5/config/packages/configuration_converter.yaml

```diff
@@ -0,0 +1,2 @@
+#configuration_converter:
+#    api_platform_default_export_path: '%kernel.project_dir%/config/packages/api-platform/' #(default)
```

Comment:
If you uncomment this I'll be happy to merge.

Reply:
Yup, thanks for your review, it's been done.